packages/sdk-js/src/feature-repository.ts

```diff
@@ -276,7 +276,7 @@ async function fetchFeatures(
 
 // Watch a feature endpoint for changes
 // Will prefer SSE if enabled, otherwise fall back to cron
-function startAutoRefresh(instance: GrowthBook) {
+function startAutoRefresh(instance: GrowthBook, errors = 0): void {
```

Comment:
Don't use default argument values, causes a lot of unnecessary code in the final build.  Set it as `errors?: number` and add a default when using it `errors || 0`.